build.gradle

```diff
@@ -73,15 +75,6 @@ subprojects {
 
     // Dependency Management - to fix versions. Pick up maven build settings for now
     dependencies {
-        constraints {
```

Comment:
we used constraints here to have consistent versions across any subprojects - in particular of the code egeria components, but also IMO any utilities. Though we only have one subproject at the moment. I will re-add this change